### ISSUE TITLE
Warning popup for non-existent directories doesn't work on Windows

### ISSUE THREAD

**Issue by Kasterisk** [Revolutionary-Games/Thrive#2589]
Was attempting to try out #2580 functionality and strangely the dialogs doesn't seem to want to popup on my end, I'm assuming it was a problem with maybe the signals or code or something but everything looks fine. So I looked up Godot's source code to see what's up and apparently `OS.ShellOpen` for Windows always returns `Ok` even if no file is found (among other errors).

From https://github.com/godotengine/godot/blob/3.3/platform/windows/os_windows.cpp#L3192-L3196:
```c++
Error OS_Windows::shell_open(String p_uri) {

	ShellExecuteW(NULL, NULL, p_uri.c_str(), NULL, NULL, SW_SHOWNORMAL);
	return OK;
}
```

Definitely seems like a bug(?), though not sure if intended, I might report this to Godot and open a new issue there sometime later.

**Comment by Kasterisk** [comment]
Opened an issue https://github.com/godotengine/godot/issues/52794.

**Comment by Kasterisk** [comment]
#2742 has been merged, we now use Godot 3.4.
